source/common/tcp_proxy/tcp_proxy.cc

```diff
@@ -850,6 +850,11 @@ void Filter::onAccessLogFlushInterval() {
     access_log->log(nullptr, nullptr, nullptr, getStreamInfo(),
                     AccessLog::AccessLogType::TcpPeriodic);
   }
+  const SystemTime now = read_callbacks_->connection().dispatcher().timeSource().systemTime();
+  getStreamInfo().getDownstreamBytesMeter()->takeDownstreamPeriodicLoggingSnapshot(now);
+  if (getStreamInfo().getUpstreamBytesMeter()) {
+    getStreamInfo().getUpstreamBytesMeter()->takeDownstreamPeriodicLoggingSnapshot(now);
```

Comment:
should not it be `takeUpstreamPeriodic...`?

Reply:
No. This is the snapshot of the Upstream bytes meter intendded for the Downstream access logger.